spec/controllers/application_controller_spec.rb

```diff
@@ -17,6 +17,13 @@ def test_action
     end
   end
 
+  describe "click_event events" do
```

Comment:
Can we try to stop writing controller specs and write request specs instead?

Reply:
Ha ha, yes we can! I just didn't want to rewrite `application_controller_spec.rb` today 😂